streams/src/test/java/org/apache/kafka/streams/kstream/KStreamBuilderTest.java

```diff
@@ -85,6 +86,19 @@ public void testNewName() {
     }
 
     @Test
+    public void testNewStoreName() {
```

Comment:
I meant to do this in one test, because the second part of the test validates that we start again from 0.